src/main/java/cs361/battleships/models/Submarine.java

```diff
@@ -16,6 +17,17 @@ public Submarine(String kind) {
 		this.kind = kind.toUpperCase();
 	}
 
+	public Submarine(Ship ship)
```

Comment:
It's a shame to have to do it this way, but I understand the need, and we did need to get this done. Don't feel too bad about getting it to work. If we had more time I would totally like fixing this up.